request.go

```diff
@@ -17,7 +17,14 @@ import (
 
 func init() {
 	goreq.SetConnectTimeout(15 * time.Second)
-	goreq.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{RootCAs: getCACerts()}
+	if !useSystemCerts() {
+		goreq.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{RootCAs: getCACerts()}
+	}
+}
+
+func useSystemCerts() bool {
+	e := os.Getenv("HEROKU_USE_SYSTEM_CERTS")
+	return e != "false" && e != "0"
```

Comment:
@dickeyxxx If HEROKU_USE_SYSTEM_CERTS is not set, this returns true.  Do not fix this just yet, I am trying something else out and should have a pull request in a bit, just thought I would mention